### PR TITLE
[docs/rust]: Fix typo

### PR DESCRIPTION
**Pull request by alexfmpe**

### CODE REVIEW

**Review by cole-h — APPROVED** [review]

**Review by wamserma — CHANGES_REQUESTED** [review]
please give your PR a better title, so one can see immediately what this is about, e.g.
"[docs/rust]: fix typo"

otherwise: thank you for spotting and fixing this

**Comment by cole-h** [comment]
(also change the commit title, please)

**Review by wamserma — APPROVED** [review]

File: doc/languages-frameworks/rust.section.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ rustPlatform.buildRustPackage rec {
 `buildRustPackage` requires a `cargoSha256` attribute which is computed over
 all crate sources of this package. Currently it is obtained by inserting a
 fake checksum into the expression and building the package once. The correct
-checksum can be then take from the failed build.
+checksum can be then taken from the failed build.
```
Comment (drewrisinger) [inline doc/languages-frameworks/rust.section.md:53]:
```suggestion
checksum can then be taken from the failed build.
```

**Review by drewrisinger — APPROVED** [review]

**Comment by danieldk** [comment]
Squashed the commits and cleanup the commit message. Thanks for the fix!

**Pull request merged**